core/server/api_container/server/startosis_engine/recipe/http_request_recipe.go

```diff
@@ -27,11 +27,11 @@ const (
 
 	statusCodeKey    = "code"
 	bodyKey          = "body"
-	extractKeyPrefix = "extract"
+	ExtractKeyPrefix = "extract"
```

Comment:
Yup, we need to migrate that one to the framework. I believe the recipe types are the only ones left